src/D2L.CodeStyle.Analyzers/Extensions/Microsoft.CodeAnalysis.cs

```diff
@@ -164,5 +164,32 @@ string name
 
 			return false;
 		}
+
+		public static ImmutableArray<IMethodSymbol> GetImplementedMethods(

```

Comment:
Possible other things that could use this helper: https://github.com/Brightspace/D2L.CodeStyle/search?q=FindImplementationForInterfaceMember